docs/modelserving/v1beta1/amd/multi_model.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: "serving.kserve.io/v1beta1"
+kind: InferenceService
+metadata:
+  annotations:
+    autoscaling.knative.dev/target: "5"
+  labels:
+    controller-tools.k8s.io: "1.0"
+    app: example-amdserver-multi-isvc
+  name: example-amdserver-multi-isvc
+spec:
+  predictor:
+    containers:
+      - name: custom
+        image: <image>
+        args:
+          - proteus-server
+          - --http-port=8080
+          - --grpc-port=9000
+        env:
+          - name: MULTI_MODEL_SERVER
+            value: "true"
+        ports:
+          - containerPort: 8080
+            protocol: TCP
```

Comment:
I think we should create the serving runtime instead of using the custom container. see Q&A [here](https://github.com/kserve/kserve/discussions/2450)